services/checks/checks.go

```diff
@@ -115,6 +118,20 @@ func New(agentsRegistry agentsRegistry, alertsRegistry alertRegistry, db *reform
 		publicKeys: defaultPublicKeys,
 		interval:   defaultInterval,
 		startDelay: defaultStartDelay,
+
+		mExecutedScripts: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: prometheusNamespace,
+			Subsystem: prometheusSubsystem,
+			Name:      "check_script_executed_total",
```

Comment:
This way the full metric name will be `pmm_managed_checks_check_script_executed_total`. I think that `checks_check` part is too mouthful. I think `"script_executed_total"` would be enough there.